GliaWidgets/CoreSDKClient/CoreSDKClient.Mock.swift

```diff
@@ -24,6 +24,15 @@ extension CoreSdkClient {
     )
 }
 
+extension CoreSdkClient.Operator {
+    static let mock = CoreSdkClient.Operator(
```

Comment:
I think it will be more beneficial to make it a function with these default values as arguments. This will allow this mock to be reusable between the tests. Does it make sense?

Reply:
fixed in 64bfe8bfa8beb92d3bce127f30d4e957b2aab486